frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -124,6 +159,7 @@ export const InteractiveEmbeddingOptionCard = () => {
         href={`${quickStartUrl}?utm_source=${plan}&utm_media=embed-settings`}
       >
         {t`Check out our Quick Start`}
+        <Icon name="share" aria-hidden />
```

Comment:
a new icon from the design 
![image](https://github.com/user-attachments/assets/86438d03-3f3e-4ea2-beab-ef573cf5a178)